src/test/java/homework17.java

```diff
@@ -22,23 +21,37 @@ public void addSongToPlaylist() {
 //String namePlaylist = UUID.randomUUID().toString();
 
 
-        WebElement search = driver.findElement(By.cssSelector("input[type='search']"));
-        search.sendKeys("F");
- WebElement brand = driver.findElement(By.cssSelector("[class='brand']"));
- brand.click();
-
-WebElement allViewButton = driver.findElement(By.cssSelector("[data-test='view-all-songs-btn']"));
-allViewButton.click();
-WebElement clickSong = driver.findElement(By.cssSelector("#songResultsWrapper > div > div > div.item-container > table > tr"));
-clickSong.click();
-WebElement addToButton = driver.findElement(By.cssSelector("[class='btn-add-to']"));
-addToButton.click();
-WebElement playlistNameNew = driver.findElement(By.xpath("//*[@id='queueWrapper']/header/div[3]/div/section[2]/form/input"));
-playlistNameNew.sendKeys("HW17");
-WebElement submitButton = driver.findElement(By.xpath("//*[@id='songResultsWrapper']/header/div[3]/div/section[2]/form/button"));
-submitButton.click();
-
-
+//        WebElement search = driver.findElement(By.cssSelector("input[type='search']"));
+//        search.sendKeys("F");
+// WebElement brand = driver.findElement(By.cssSelector("[class='brand']"));
+// brand.click();
+//
+//WebElement allViewButton = driver.findElement(By.cssSelector("[data-test='view-all-songs-btn']"));
+//allViewButton.click();
+//WebElement clickSong = driver.findElement(By.cssSelector("#songResultsWrapper > div > div > div.item-container > table > tr"));
+//clickSong.click();
+//WebElement addToButton = driver.findElement(By.cssSelector("[class='btn-add-to']"));
+//addToButton.click();
+//WebElement playlistNameNew = driver.findElement(By.xpath("//*[@id='queueWrapper']/header/div[3]/div/section[2]/form/input"));
+//playlistNameNew.sendKeys("HW17");
+//WebElement submitButton = driver.findElement(By.xpath("//*[@id='songResultsWrapper']/header/div[3]/div/section[2]/form/button"));
+//submitButton.click();
+//
```

Comment:
**I recommend not using code comments**: for, example, when you have a code that cannot be understood, then comments are not the solution. Try to rewrite the code or rename fields and other elements in a way that the reader will understand the action that you are doing there.